docs/src/componentDocs/ThreeLiner/examples/ThreeLinerWithCustomContent.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Box } from '@mui/material';
+import { CodeBlock, CodeBlockActionButtonRow } from '../../../shared';
+import * as colors from '@brightlayer-ui/colors';
+import { ThreeLinerWithCustomContentExample } from './ThreeLinerWithCustomContentExample';
+
+const codeSnippet = `<ThreeLiner
+    title="title"
+    subtitle="subtitle"
+    info={
+        <ToolbarMenu
```

Comment:
```suggestion
       <ToolbarMenu
            label="info"
            menuGroups={[
                {
                    items: [
                        {
                            title: 'Menu Item 1'
                        },
                        {
                            title: 'Menu Item 2'
                        },
                        {
                            title: 'Menu Item 3'
                        },
                    ],
                },
            ]}
        />
```

Reply:
@bkarambe Higlighling is breaking. You need to update data-line attr.
<img width="897" alt="Screenshot 2022-09-28 at 2 12 33 PM" src="https://user-images.githubusercontent.com/10433274/192732875-9349c94a-53de-456b-a7a6-46a8a9955fca.png">